basic-auth.js

```diff
@@ -0,0 +1,16 @@
+const auth = require('basic-auth')
+
+// You can pass multiple users if you need to,
+// store them as a keyed values in the users var
+const users = {}
+const username = process.env.USERNAME || 'demo'
```

Comment:
Does the fallback to `demo` actually do anything since this won't get used at all if there is no `env.USERNAME`?

Reply:
keeping it as is, in case somebody wants this in the future.